tardis/model/tests/test_base.py

```diff
@@ -12,13 +12,13 @@
 def data_path(filename):
     return os.path.abspath(os.path.join("tardis/io/tests/data/", filename))
 
-
 class TestModelFromPaper1Config:
+    @pytest.fixture(scope="class", autouse=True)
     def setup(self):
         filename = "paper1_tardis_configv1.yml"
-        self.config = Configuration.from_yaml(data_path(filename))
-        self.model = Radial1DModel.from_config(self.config)
-
+        type(self).config = Configuration.from_yaml(data_path(filename))
+        type(self).model = Radial1DModel.from_config(self.config)
+    
```

Comment:
This is just for the time being, I am trying to find a way to move this to a fixture(and probably to `conftest.py`).